### PR TITLE
GRPC V2 : Get Passive Delegators

### PR DESCRIPTION
**Pull request by parv0888**
## Purpose
Added support for GRPC V2 : Get Passive Delegators

## Changes
Added support for GRPC V2 : Get Passive Delegators

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [x] I accept the above linked CLA.

### CODE REVIEW

File: concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetPassiveDelegators.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.concurrent.Callable;
+
+@Command(name = "GetPassiveDelegators", mixinStandardHelpOptions = true)
+public class GetPassiveDelegators implements Callable<Integer> {
+    @Option(
+            names = {"--endpoint"},
+            description = "GRPC interface of the node.",
+            defaultValue = "http://localhost:20002")
```
Comment (abizjak) [inline concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetPassiveDelegators.java:21]: I'd recommend using 20000

File: concordium-sdk/src/main/java/com/concordium/sdk/responses/DelegatorInfo.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import java.util.Optional;
+
+@Builder
+@Getter
+@EqualsAndHashCode
+@ToString
+public class DelegatorInfo {
+    private final AccountAddress account;
+    private final CCDAmount stake;
+    private final Optional<PendingChange> pendingChange;
```
Comment (abizjak) [inline concordium-sdk/src/main/java/com/concordium/sdk/responses/DelegatorInfo.java:20]: Could you please document the fields here. I assume documentation then gets propagated to the gettters.

File: concordium-sdk/src/test/java/com/concordium/sdk/ClientV2GetPassiveDelegatorsTest.java [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+import java.time.ZoneOffset;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.AdditionalAnswers.delegatesTo;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ClientV2GetPassiveDelegatorsTest {
```
Comment (abizjak) [inline concordium-sdk/src/test/java/com/concordium/sdk/ClientV2GetPassiveDelegatorsTest.java:34]: Can you please document what the intention and scope of these tests are. It's somewhat hard to udnerstand what the intention is exactly.

File: concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetPassiveDelegators.java (hunk 1 above)
Comment (parv0888) [reply concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetPassiveDelegators.java:21]: done

File: concordium-sdk/src/main/java/com/concordium/sdk/responses/DelegatorInfo.java (hunk 2 above)
Comment (parv0888) [reply concordium-sdk/src/main/java/com/concordium/sdk/responses/DelegatorInfo.java:20]: done

File: concordium-sdk/src/test/java/com/concordium/sdk/ClientV2GetPassiveDelegatorsTest.java (hunk 3 above)
Comment (parv0888) [reply concordium-sdk/src/test/java/com/concordium/sdk/ClientV2GetPassiveDelegatorsTest.java:34]: done

**Pull request merged**